NOTES.md

```diff
@@ -1,3 +1,185 @@
+# Release 2.1.0-beta (Forbidden Planet)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-14 21:00 UTC
+-- contributors " " next as of 3dac994 and docs:master as of 0b661c4
+
+This is a beta release for RethinkDB 2.1. **It is not for production use and has known
+bugs.** Please do not use this version for production data.
+
+We are looking forward to your bug reports on [GitHub][github-issues]
+or on our [mailing list][mailing-list].
+
+[github-issues]: http://github.com/rethinkdb/rethinkdb/issues/
+[mailing-list]: https://groups.google.com/forum/#!forum/rethinkdb
+
+Release highlights:
+
+* Automatic failover using a Raft-based protocol
+* More flexible administration for servers and tables
+* Advanced recovery features
+
+Read the [blog post][2.1-beta] for more details.
+
+[2.1-beta]: http://rethinkdb.com/blog/2.1-beta/
+
+## Compatibility ##
+
+This beta release does not include automatic migration of data
+directories from older versions of RethinkDB. The final release of RethinkDB 2.1 will
+automatically migrate data from RethinkDB 1.14 and up.
+
+If you're upgrading directly from RethinkDB 1.13 or earlier, you will need to manually
+upgrade using `rethinkdb dump`.
+
+### Changed handling of server failures ###
+This release introduces a new system for dealing with server failures and network
+partitions based on the Raft consensus algorithm.
+
+Previously, unreachable servers had to be manually removed from the cluster in order to
+restore availability. RethinkDB 2.1 can resolve many cases of availability loss
+automatically, and keeps the cluster in an administrable state even while servers are
+missing.
+
+There are three important scenarios in RethinkDB 2.1 when it comes to restoring the
+availability of a given table after a server failure:
+
+* The table has three or more replicas, and a majority of the servers that are hosting
+  these replicas are connected. RethinkDB 2.1 automatically elects new primary replicas
+  to replace unavailable servers and restore availability. No manual intervention is
+  required, and data consistency is maintained.
+* A majority of the servers for the table are connected, regardless of the number of
+  replicas. The table can be manually reconfigured using the usual commands, and data
+  consistency is always maintained.
+* A majority of servers for the table are unavailable. The new `emergency_repair` option
+  to `table.reconfigure` can be used to restore table availability in this case.
+
+
+### System table changes ###
+To reflect changes in the underlying cluster administration logic, some of the tables in
+the `rethinkdb` database changed.
+
+####Changes to `table_config`:####
```

Comment:
The way that GitHub renders these means that they're enormous and hard to read. Can we change these to just be bold?

> ### System table changes
> #### Changes to `table_config`:
> __Changes to `table_config`:__